waffle_hub/hub/adapter/tx_model/tx_model_hub.py

```diff
@@ -128,13 +134,41 @@ def get_preprocess(self, task: str, *args, **kwargs):
             def preprocess(x, *args, **kwargs):
                 return normalize(x)
 
+        elif task == "classification":
+            normalize = T.Normalize([0, 0, 0], [1, 1, 1], inplace=True)
+
+            def preprocess(x, *args, **kwargs):
+                return normalize(x)
+
         return preprocess
 
     def get_postprocess(self, task: str, *args, **kwargs):
 
         if task == "object_detection":
 
+            def inner(
```

Comment:
It seems copy postprocess from ultralytics hub. Is it correct?

Reply:
As you said, the code before the change was supposed to divide the bbox by the height and width of the image like Ultralytics. However, I noticed that the output of tx model detector contains bbox[x, y, w, h] in the range of 0~1, confidence score, and class ids, so I changed it to no longer divide the bbox by the height and width of the image.